src/merge_sort.py

```diff
@@ -0,0 +1,61 @@
+""" This module will implement a merge sort on a list"""
+# This solution was heavily influenced by the example found on interactive
+# python found at the URL below. I've attempted to make the function a class
+# with a recursive method:
+# http://interactivepython.org/runestone/static/pythonds/SortSearch/TheMergeSort.html
+import timeit
+import random
+
+
+class MergeSort(object):
```

Comment:
What's the idea behind putting your merge sort function on a class definition? Bad call here, because it means a user has to create an instance of the MergeSort class in order to use the merge_sort method. Your method does not use any advantage of class-based object oriented programming, so why bother? Best to scrap the class approach, IMHO.